ql/models/equity/hestonmodel.cpp

```diff
@@ -34,6 +34,8 @@ namespace QuantLib {
                                           BoundaryConstraint(-1.0, 1.0));
         arguments_[4] = ConstantParameter(process->v0(),
                                           PositiveConstraint());
+        arguments_[5] = ConstantParameter(process->mixingFactor(),
```

Comment:
Is this going to calibrate the mixing factor as well?  If so, it should be kept out of the vector of arguments.

Reply:
IMHO we should consider adding the mixing parameter to the HestonSLVProcess not to the HestonProcess and HestonModel. The Heston process a such does not have a mixing factor and it might be misleading. I'd suggest to remove the mixingFactor from the HestonModel. In the current form it will distort the calibration of any Heston models and it has unwanted side effects for the BatesModel.

If we add the mixingFactor to the HestonSLVProcess we can either define a new constructor
for the FdHestonVanillaEngine or introduce FdHestonSlvVanillaEngine, which is just a proxy for the FdHestonVanillaEngine with the right constructor.

Reply:
Done. For the constructor in FdHestonVanillaEngine, I added mixingFactor as a default parameter such that it defaults to value of 1.0 if that value is no supplied.